apps/artwork/components/auction/test/view.coffee

```diff
@@ -111,6 +112,25 @@ describe 'auction', ->
         @view.$('.artwork-auction__buy-now')
           .should.have.lengthOf 0
 
+    describe 'reserve not met', ->
+      it 'gives a reserve not met bid message', ->
+        data = Object.assign({}, @data)
+        data.accounting = accounting
+        data.artwork.sale_artwork.reserve_message = 'Reserve not met'
+        # just shoving user + me into data to meet the template conditions
+        # to make .is-losing message render
+        data.user = true
+        data.me =
+          lot_standing:
+            is_highest_bidder: false
+            most_recent_bid:
+              max_bid:
+                cents: 5500000
+        view = new @ArtworkAuctionView data: data
```

Comment:
Nice reconstructing in this test! It's possible you could just mutate `@data` and run `view.render()` to get the same result (just guessing), but this is a good strategy to have regardless.